.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
-. "$(dirname "$0")/_/husky.sh"
+# #!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
 
-yarn lint-staged
+yarn run format-files && yarn run tsc && yarn lint:all
```

Comment:
new line, please add it ;-) 

Not loking forward to long tsc linting, but probably it's for the best